microcosm_flask/swagger/parameters/enum.py

```diff
@@ -59,10 +59,9 @@ def parse_type(self, field: Field) -> str:
             raise Exception(f"Cannot infer enum type for field: {field.name}")
 
     def _parse_enum_values(self, field: Field) -> Sequence:
-        enum = getattr(field, "enum", None)
         return [
             choice.value if field.by_value else choice.name  # type: ignore
-            for choice in enum
+            for choice in field.enum
```

Comment:
Unrelated change to fix typehinting. Not that in the `None` case line 63 would have errored anyway, so this change doesn't introduce a new risk. This code is only reached when self.strict_enums is true (see line 69), in which case the "enum" attribute should be present.